elasticsearch/Dockerfile.centos7

```diff
@@ -0,0 +1,52 @@
+FROM centos:centos7
```

Comment:
Can we generate the Dockefile from a template that is given the proper parameter?